sources/packages/backend/workflow/test/2021-2022/assessment-gateway/assessment-gateway.originalAssessment.e2e-spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ASSESSMENT_ID } from "@sims/services/workflow/variables/assessment-gateway";
+import {
+  ApplicationExceptionStatus,
+  AssessmentTriggerType,
+  ProgramInfoStatus,
+} from "@sims/sims-db";
+import { ZBClient } from "zeebe-node";
+import {
+  createFakeConsolidatedFulltimeData,
+  ZeebeMockedClient,
+  E2E_STUDENT_STATUS,
+  E2E_APPLICATION_EXCEPTION_STATUS,
+  E2E_PIR_STATUS,
+  PROCESS_INSTANCE_CREATE_TIMEOUT,
+  Workers,
+} from "../../test-utils";
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+
+describe(`E2E Test Workflow assessment gateway on original assessment for ${PROGRAM_YEAR}`, () => {
+  let zeebeClientProvider: ZBClient;
+  beforeAll(async () => {
+    zeebeClientProvider = ZeebeMockedClient.getMockedZeebeInstance();
+  });
+
+  it("Should follow the expected workflow path when student single and independent without application exception and PIR.", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.assessmentTriggerType =
+      AssessmentTriggerType.OriginalAssessment;
+
+    // Act/Assert
+    const assessmentGatewayResponse =
+      await zeebeClientProvider.createProcessInstanceWithResult({
+        bpmnProcessId: "assessment-gateway",
+        variables: {
+          ...assessmentConsolidatedData,
+          [ASSESSMENT_ID]: 1,
+          // Based on this variable, load consolidated data worker will
+          // provide mock value for student dependent status and relationship status.
+          [E2E_STUDENT_STATUS]: "independentSingleStudent",
+          // Based on this variable load consolidated data worker will
+          // return the application exception status.
+          [E2E_APPLICATION_EXCEPTION_STATUS]:
+            ApplicationExceptionStatus.Approved,
+          // Based on this variable load consolidated data worker will
+          // return the application PIR status.
+          [E2E_PIR_STATUS]: ProgramInfoStatus.notRequired,
+        },
+        requestTimeout: PROCESS_INSTANCE_CREATE_TIMEOUT,
+      });
+    // Workflow instance expected to pass through associate workflow instance worker.
+    expect(
+      assessmentGatewayResponse.variables[Workers.AssociateWorkflowInstance],
```

Comment:
I would not mind having these ones created at the `lib` level and shared between the worker's application also.
```ts
@ZeebeWorker(Workers.AssociateWorkflowInstance
```